dpnp/random/_random.pyx

```diff
@@ -300,114 +300,114 @@ cpdef dparray dpnp_gumbel(double loc, double scale, size):
     return result
 
 
-cpdef dparray dpnp_negative_binomial(double a, double p, size):
+cpdef dparray dpnp_laplace(double loc, double scale, size):
     """
-    Returns an array populated with samples from negative binomial distribution.
-
-    `negative_binomial` generates a matrix filled with random floats sampled from a
-    univariate negative binomial distribution for a given parameter of the distribution
-    `a` and success probability `p` of a single trial.
+    Returns an array populated with samples from beta distribution.
+    `dpnp_laplace` generates a matrix filled with random floats sampled from a
+    univariate laplace distribution.
 
     """
 
-    dtype = numpy.int32
+    dtype = numpy.float64
```

Comment:
@samir-nasibli Please use DPNP specific types as much as possible
https://github.com/IntelPython/dpnp/blob/4664aac456d8d20f6f115292ea1ccabf3af403d9/dpnp/dpnp_iface_types.py#L46

Reply:
I see other same issues below

Reply:
If it is critical, I can change it on other PR.